### ISSUE TITLE
Add ability to Share and export a note . 

### ISSUE THREAD

**Issue by mad1mystic** [anyproto/anytype-kotlin#269]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing requests and didn't find any that were similar
- [X] I have considered creating a pull request instead and want to proceed

### Clear and concise description of the problem

Not able to share or export a Note on Android app. How is that an app don't have the ability to Share or export? Ability to create encrypted backup on phone . Ability to encrypt on certain notes. Ability to Share and export in different format like md or pdf 

### Suggested solution

Share via Android menu and or export a note . To have ability to Share or export in different format like md or pdf

**Comment by devkamiki** [comment]
yeah i think this is really a good feature, to export to pdf or markdown just like on desktop;)
